### PR TITLE
chore: bump marked  version from 0.8.0 to 7.0.5

### PR DESCRIPTION
**Pull request by JerrysShan**
upgrade deps version  ，because of  [security vulnerabilities](https://security.snyk.io/vuln/npm?search=marked)

### CODE REVIEW

**Comment by JerrysShan** [comment]
cc @cameronhunter

**Comment by cameronhunter** [comment]
Thanks for the contribution!

File: yarn.lock [hunk 1]
```diff
@@ -2370,10 +2370,10 @@ marked-terminal@^4.0.0:
     node-emoji "^1.10.0"
     supports-hyperlinks "^2.0.0"
 
-marked@^0.8.0:
-  version "0.8.0"
-  resolved "https://registry.yarnpkg.com/marked/-/marked-0.8.0.tgz#ec5c0c9b93878dc52dd54be8d0e524097bd81a99"
-  integrity sha512-MyUe+T/Pw4TZufHkzAfDj6HarCBWia2y27/bhuYkTaiUnfDYFnCP3KUN+9oM7Wi6JA2rymtVYbQu3spE0GCmxQ==
+marked@^7.0.5:
+  version "7.0.5"
+  resolved "http://bnpm.byted.org/marked/-/marked-7.0.5.tgz#8a9e4e3afb93b58fe9ee7608e67cc154eb15d508"
```
Comment (cameronhunter) [inline yarn.lock:2375]: Hmmmm, why not from the yarn registry?

**Comment by cameronhunter** [comment]
I've fixed the issue in `yarn.lock` and I've published `v1.0.4` that includes this pull request. Thanks for the contribution!

**Pull request merged**